android/src/firebase/config/TitaniumFirebaseConfigModule.java

```diff
@@ -4,148 +4,186 @@
+                    result.put(PROPERTY_SUCCESS, true);
+                    result.put(PROPERTY_RESULT, task.getResult());
+                } else {
+                    result.put(PROPERTY_SUCCESS, false);
+                    if (task.getException() != null) {
+                        result.put(PROPERTY_ERROR, task.getException().getMessage());
+                    }
+                }
+                callback.callAsync(getKrollObject(), result);
+            }
+        });
+    }
+
+    @Kroll.method
+    public void setMinimumFetchIntervalInSeconds(int minimumFetchInterval) {
+        FirebaseRemoteConfigSettings configSettings = new FirebaseRemoteConfigSettings.Builder()
+                .setMinimumFetchIntervalInSeconds(minimumFetchInterval)
+                .build();
+        FirebaseRemoteConfig.getInstance().setConfigSettingsAsync(configSettings);
+    }
+
+    @Kroll.method
+    public void setDefaults(Object params) throws Exception {
+        if (params instanceof String) {
+            FirebaseRemoteConfig.getInstance().setDefaultsAsync(TiRHelper.getResource("xml." + TiConvert.toString(params)));
+        } else if (params instanceof HashMap) {
+            FirebaseRemoteConfig.getInstance().setDefaultsAsync((HashMap) params);
+        } else {
+            throw new Exception("Invalid defaults provided. Please either pass a dictionary or string");
+        }
+    }
+
+    @Kroll.method
+    public void activateFetched(KrollFunction callback) {
+        FirebaseRemoteConfig.getInstance().activate().addOnCompleteListener(task -> {
+            if (callback != null) {
+                KrollDict result = new KrollDict();
+                if (task.isSuccessful()) {
+                    result.put(PROPERTY_SUCCESS, true);
+                    result.put(PROPERTY_RESULT, task.getResult());
+                } else {
+                    result.put(PROPERTY_SUCCESS, false);
+                    if (task.getException() != null) {
+                        result.put(PROPERTY_ERROR, task.getException().getMessage());
+                    }
+                }
+                callback.callAsync(getKrollObject(), result);
+            }
+        });
+    }
+
+    @Kroll.method
+    public KrollDict configValueForKey(String key) {
+        String stringValue = FirebaseRemoteConfig.getInstance().getString(key);
+        boolean boolValue = FirebaseRemoteConfig.getInstance().getBoolean(key);
+        double numberValue = FirebaseRemoteConfig.getInstance().getDouble(key);
+
+        KrollDict result = new KrollDict();
+        result.put("bool", boolValue);
+        result.put("number", numberValue);
+
+        if (stringValue != null) {
+            result.put("string", stringValue);
+        }
+
+        return result;
+    }
+
+    @Kroll.setProperty
+    public void enableRealtimeUpdates(Boolean value) {
+        if (value) {
+			culr = FirebaseRemoteConfig.getInstance().addOnConfigUpdateListener(new ConfigUpdateListener() {
```

Comment:
Odd formatting. This should not happen after a linting. Maybe added later? Also, use a more expressive name, e.g. `configUpdateListener`